tests/test_benchmark.py

```diff
@@ -111,7 +155,12 @@ def test_train_encoder_decoder_with_configs(self):
         MODEL_ID = "sshleifer/tinier_bart"
         config = AutoConfig.from_pretrained(MODEL_ID)
         benchmark_args = PyTorchBenchmarkArguments(
-            models=[MODEL_ID], training=True, no_inference=True, sequence_lengths=[8], batch_sizes=[1]
+            models=[MODEL_ID],
+            training=True,
+            no_inference=True,
+            sequence_lengths=[8],
+            batch_sizes=[1],
+            no_multi_process=True,
```

Comment:
multiprocess and pytest are not friends it seems?

Reply:
nope :D PyTest also starts a new process in itself for each test so multiprocessing in multiprocessing breaks CUDA inits.